### PR TITLE
Route to the first search result upon pressing enter

### PR DESCRIPTION
**Pull request by marcoambrosini**
Sneaky lil change, been annoyed by this for too long

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/LeftSidebar/LeftSidebar.vue [hunk 1]
```diff
@@ -490,6 +491,14 @@ export default {
 			this.handleScroll()
 			this.handleUnreadMention()
 		}, 50),
+
+		// Route to the first search result
+		handleEnter() {
+			if (this?.conversationsList[0]?.token) {
+				this.$router.push({ name: 'conversation', params: { token: this.conversationsList[0].token } })
```
Comment (nickvergessen) [inline src/components/LeftSidebar/LeftSidebar.vue:498]: return after this?

File: src/components/LeftSidebar/LeftSidebar.vue [hunk 2]
```diff
@@ -490,6 +491,14 @@ export default {
...
+			}
+			this.handleClickSearchResult()
```
Comment (nickvergessen) [inline src/components/LeftSidebar/LeftSidebar.vue:500]: This should start a 1-1 if the first shown entry is a user?
Doesn't work for me

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/LeftSidebar/LeftSidebar.vue (hunk 1 above)
Comment (marcoambrosini) [reply src/components/LeftSidebar/LeftSidebar.vue:498]: Why? There's no need to return anything there

Comment (nickvergessen) [reply src/components/LeftSidebar/LeftSidebar.vue:498]: Okay, seems I misunderstood the part after the if

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**